Packages/manifest.json

```diff
@@ -1,12 +1,18 @@
 {
   "dependencies": {
     "com.unity.ads": "2.3.1",
-    "com.unity.analytics": "3.2.2",
-    "com.unity.collab-proxy": "1.2.15",
-    "com.unity.package-manager-ui": "2.0.3",
-    "com.unity.purchasing": "2.0.3",
-    "com.unity.textmeshpro": "1.4.1",
-    "com.unity.xr.openvr.standalone": "1.0.2",
+    "com.unity.analytics": "3.3.2",
+    "com.unity.collab-proxy": "1.2.16",
+    "com.unity.multiplayer-hlapi": "1.0.2",
+    "com.unity.package-manager-ui": "2.1.2",
+    "com.unity.purchasing": "2.0.6",
```

Comment:
ARFoundation related items may need to be added but the others should be removed